### PR TITLE
0x4d - Updates, fixes and editing

### PR DESCRIPTION
**Pull request by romualdszkudlarek**

### CODE REVIEW

File: Document/0x04d-Security-Testing-SDLC.md [hunk 1]
```diff
@@ -38,29 +38,44 @@ From the human aspect, this is achieved by creating cross functional teams that
...
+- At the beginning of a project or a development cycle, at the same time when functional requirements are gathered, **Security Requirements** are listed and clarified. As use cases are built, **Abuse Cases** are added. Teams (including development teams) may be trained on security if needed (Secure Coding, ...);
+- For mobile applications, the OWASP MASVS (https://www.owasp.org/images/f/fe/MASVS_v0.9.3.pdf) can be leveraged to determine the security requirements based on the risk assessment that was conducted in this initial step. It is common, especially for agile projects, to iteratively review the set of requirements based on newly added features and new classes of data that is handled by the application.
```
Comment (Sjord) [inline Document/0x04d-Security-Testing-SDLC.md]: Please use a inline markdown link: `[text](link "title")`

File: Document/0x04d-Security-Testing-SDLC.md [hunk 2]
```diff
@@ -38,29 +38,44 @@ From the human aspect, this is achieved by creating cross functional teams that
...
+- All security requirements and design considerations should be stored in the Application Life cycle Management System (ALM), which is typically known as issue tracker, that the development/ops team already uses to ensure that security requirements are tightly integrated into the development workflow. The security requirements should ideally also contain the relevant source code snippets for the used programming language, to ensure that developers can quickly reference them. Another strategy for secure coding guidelines is to create a dedicated repository under versioning control, that only contains these code snippets, which has many benefits over the traditional approach of storing these guidelines in word documents or PDFs.
+- The next step is to **develop software**, including security activities such as **Code Reviews** (usually with peers), **Static Analysis** with automated tools and **Unit Tests** dedicated to security;
```
Comment (Sjord) [inline Document/0x04d-Security-Testing-SDLC.md]: > security activities

I understand that you want to indicate the relation between these actions and security, but I don't think code reviews and static analysis are "security activities". Maybe a separate sentence would be better, where you indicate the importance of these things for security.

File: Document/0x04d-Security-Testing-SDLC.md [hunk 3]
```diff
@@ -38,29 +38,44 @@ From the human aspect, this is achieved by creating cross functional teams that
...
 
 The picture below shows all the phases with the different artefacts:
 -- TODO [Add a picture of a SDLC diagram that clarifies the description above] --
 
-Based on the risks of the project, some artefacts may be simplified (or even skipped) while others may be added (formal intermediary approvals, documentation of certain points, ...). **Always keep in mind a SDLC is meant to bring risk reduction to software development and is a framework that helps put in place controls that will reduce those risks to an acceptable level. ** While this is a generic description of SDLC, always tailor this framework to the needs of your projects.
+Based on the risks of the project, some artefacts may be simplified (or even skipped) while others may be added (formal intermediary approvals, formal documentation of certain points, ...). **Always keep in mind a SDLC is meant to bring risk reduction to software development and is a framework that helps put in place controls that will reduce those risks to an acceptable level. ** While this is a generic description of SDLC, always tailor this framework to the needs of your projects.
```
Comment (sushi2k) [inline Document/0x04d-Security-Testing-SDLC.md]: Can you please fix the formatting? The rest of the chapter is bold. The "**" should come directly after "level" or "projects" without a space.

File: Document/0x04d-Security-Testing-SDLC.md [hunk 4]
```diff
@@ -38,29 +38,44 @@ From the human aspect, this is achieved by creating cross functional teams that
...
+
+##### Release phase
+
+At this point of time, code has been created and tested. Its security level has been assessed; often, metrics are produced to support evidence that code meets the expected level of security. However, it now has to be transitioned to the Customer, e.g. it has to be accepted by stakeholders (Management, Marketing, ...) as able to create value on the market and be of economical interest to Customers; next to that, it will be made available to the market. It is not enough to produce secure software, but it now has to be safely transitioned to Production environments, which in turn must be secured (both in the short term and in the long term); documentation for Operation teams may be created. In this phase, stakeholders (Management, Marketing, ...) are first involved, as well as technical teams (Testing, Operations, Quality, ...).
```
Comment (sushi2k) [inline Document/0x04d-Security-Testing-SDLC.md]: Some words in the PR shouldn't be uppercase like "Customer", "Proof of Concepts", "Management, "Marketing", "Testing" etc. Can you please make the first letter lowercase?

File: Document/0x04d-Security-Testing-SDLC.md (hunk 2 above)
Comment (romualdszkudlarek) [inline Document/0x04d-Security-Testing-SDLC.md]: In my mind, Security Code Reviews and Static Code Analysis are part of security. Maybe the confusing thing is that they can also be considered as "general" development activities. I would agree with this, however they also have their equivalent in security (hence the "dedicated to security"). Maybe this point need to be clarified, I will propose a different phrasing.

File: Document/0x04d-Security-Testing-SDLC.md [hunk 5]
```diff
@@ -40,17 +40,17 @@ Whatever the development methodology that is being used, a SDLC always follows t
...
 - At the beginning of a project or a development cycle, at the same time when functional requirements are gathered, **Security Requirements** are listed and clarified. As use cases are built, **Abuse Cases** are added. Teams (including development teams) may be trained on security if needed (Secure Coding, ...);
-- For mobile applications, the OWASP MASVS (https://www.owasp.org/images/f/fe/MASVS_v0.9.3.pdf) can be leveraged to determine the security requirements based on the risk assessment that was conducted in this initial step. It is common, especially for agile projects, to iteratively review the set of requirements based on newly added features and new classes of data that is handled by the application.
+- For mobile applications, the [OWASP MASVS] (https://www.owasp.org/images/f/fe/MASVS_v0.9.3.pdf "OWASP MASVS") can be leveraged to determine the security requirements based on the risk assessment that was conducted in this initial step. It is common, especially for agile projects, to iteratively review the set of requirements based on newly added features and new classes of data that is handled by the application.
```
Comment (Sjord) [inline Document/0x04d-Security-Testing-SDLC.md]: The space between the `]` and the `(` prevents this link from being formatted correctly. Please remove the space.

**Comment by Sjord** [comment]
Other than that, this looks good to me. 👍

**Comment by romualdszkudlarek** [comment]
Space removed

**Comment by sushi2k** [comment]
Thanks @romualdszkudlarek

**Pull request merged**